.github/workflows/coverage.yml

```diff
@@ -6,9 +6,9 @@ on:
     branches:
       - main
     tags:
-      - '*'
+      - "*"
```

Comment:
this is already a huge PR, since you're not really doing anything here, can we remove this change?